src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -343,18 +343,19 @@ const MetaTraderConfig = (() => {
 
     const fields = {
         new_account: {
-            txt_name         : { id: '#txt_name',          request_field: 'name' },
             txt_main_pass    : { id: '#txt_main_pass',     request_field: 'mainPassword' },
             txt_re_main_pass : { id: '#txt_re_main_pass' },
             ddl_trade_server : { id: '#ddl_trade_server', is_radio: true },
             chk_tnc          : { id: '#chk_tnc' },
             additional_fields: acc_type => {
                 const sample_account = getSampleAccount(acc_type);
                 const is_demo = /^demo_/.test(acc_type);
+                const get_settings = State.getResponse('get_settings');
                 return ({
                     account_type: is_demo ? 'demo' : sample_account.market_type,
                     email       : Client.get('email'),
                     leverage    : sample_account.leverage,
+                    name        : `${get_settings.first_name} ${get_settings.last_name}`,
```

Comment:
This can be empty when client doesn't have a account.

Reply:
what do you mean doesn't have an account?

Reply:
in binary metatrader is a logged in page. this is the API request sent on clicking create account button.

Reply:
Oh I forgot to mention "Real" account.